src/smtp/SMTPServer.ts

```diff
@@ -75,6 +84,18 @@ export default class SMTPServer {
 				}
 				const email = msg.split(":")[1].split(">")[0].replace("<", "")
```

Comment:
```suggestion
				const email = msg.substring(msg.indexOf("<") + 1, msg.lastIndexOf(">"))
```
should work
test first though